content/attacks/posts/2023-05-01-Level-Finance.md

```diff
@@ -0,0 +1,34 @@
+---
+date: 2023-05-01
+type of attack: "Protocol exploit"
+target entity: "Level Finance"
+type of entity: "Blockchain"
```

Comment:
Rewrite as 
>target-entities: Silk Road 
entity-types: 
    - Darknet
    - Market
attack-types: 
    - Web Infrastructure Attack
    - Race Condition Exploit

Reply:
Type of Entities for Level Finance are: DeFi, Exchange